core/library.go

```diff
@@ -409,7 +409,7 @@ func (l *library) GetExportedVariables(ctx blueprint.ModuleContext) (expLocalInc
 		if ctx.OtherModuleDependencyTag(dep) == wholeStaticDepTag ||
 			ctx.OtherModuleDependencyTag(dep) == staticDepTag ||
 			ctx.OtherModuleDependencyTag(dep) == sharedDepTag ||
-			ctx.OtherModuleDependencyTag(dep) == flagDepTag {
+			ctx.OtherModuleDependencyTag(dep) == reexportLibsTag {
```

Comment:
OK, so in this commit you've also clarified/renamed the `reexport_libs` fields. Can you mention that in the commit message please?

Reply:
I think this is still not done?